milvus/client/pool.py

```diff
@@ -9,7 +9,6 @@
 
 from . import __version__
 from .grpc_handler import GrpcHandler, RegistryHandler
-from .http_handler import HttpHandler
```

Comment:
Are we not going to support Http request?

Reply:
These codes are outdated. And we will not support HTTP in near future.